docs/src/installation.md

```diff
@@ -72,6 +72,19 @@ Such solvers include [Bonmin](https://github.com/coin-or/Bonmin) and
 [Couenne](https://github.com/coin-or/Couenne). See a more complete list
 [here](https://ampl.com/products/solvers/all-solvers-for-ampl/).
 
+Additionally, you may use [GAMS](https://github.com/GAMS-dev/gams.jl) to access additional solvers. Among them are 
+[AlphaECP](https://www.gams.com/latest/docs/S_ALPHAECP.html), 
+[Antigone](https://www.gams.com/latest/docs/S_ANTIGONE.html), 
+[BARON](https://www.gams.com/latest/docs/S_BARON.html), 
+[CONOPT](https://www.gams.com/latest/docs/S_CONOPT.html), 
+[Couenne](https://www.gams.com/latest/docs/S_COUENNE.html), 
```

Comment:
I suppose Couenne is better used through AmplNLWriter (https://github.com/jump-dev/AmplNLWriter.jl)?

Reply:
We should definitely recommend it through GAMS as well. The more the merrier.

Reply:
I don't see an issue with listing Couenne here, especially since the GAMS interface has commercial support whereas the AmplNLWriter interface does not.